### ISSUE TITLE
Unit tests failing on Node v0.12

### ISSUE THREAD

**Issue by Krinkle** [gruntjs/grunt-contrib-copy#216]
Commit https://github.com/gruntjs/grunt-contrib-copy/commit/16c2dfa1ea72 added Travis CI matrixes for Node v0.12 (and iojs). However they're currently failing:

```
Running "nodeunit:tests" (nodeunit) task
Testing copy_test.js......F
>> copy - timestamp
>> Error: 1424419216000 == 1424419216372
>> at Object.exports.copy.timestamp (test/copy_test.js:81:10)
>> copy - timestamp
>> Error: 1424419216000 == 1424419216372
>> at Object.exports.copy.timestamp (test/copy_test.js:82:10)
```
```js
/*81*/    test.equal(fs.lstatSync('tmp/copy_test_timestamp/sub_folder').mtime.getTime(), fs.lstatSync('test/fixtures/time_folder/sub_folder').mtime.getTime());
/*82*/    test.equal(fs.lstatSync('tmp/copy_test_timestamp/test.js').mtime.getTime(), fs.lstatSync('test/fixtures/time_folder/test.js').mtime.getTime());

```

Looks like timestamps are of different accuracy (not including milliseconds).

**Comment by shama** [comment]
Looks related to https://github.com/nodejs/node-v0.x-archive/issues/7000

**Comment by nalajcie** [comment]
I've created pull request (https://github.com/gruntjs/grunt-contrib-copy/pull/268) which fixes this on linux platform. Unfortunately this issue is unfixable on windows (due to bug in libuv - details in the pull request).

**Comment by XhmikosR** [comment]
Fixed with #268.

**Comment by vladikoff** [comment]
Yay!
